src/components/info-tile/info-tile.tsx

```diff
@@ -183,10 +183,14 @@ export class InfoTile {
     private renderValue = () => {
         const characterCount = (this.value ?? '').toString().length;
 
-        if (!this.value && this.loading) {
+        if ((this.value === 'undefined' || !this.value) && this.loading) {
```

Comment:
I don't think we should have this check. `this.value` should not get the value `undefined` in the first place so the bug is somewhere else :slightly_smiling_face: